internal/core/data/v2/application/event.go

```diff
@@ -242,3 +242,15 @@ func EventsByTimeRange(start int, end int, offset int, limit int, dic *di.Contai
 	}
 	return events, nil
 }
+
+// ReadingTotalCount return the count of all of readings currently stored in the database and error if any
+func ReadingTotalCount(dic *di.Container) (uint32, errors.EdgeX) {
```

Comment:
Move to new `reading.go` since focus is on Reading, not Event. Other Reading APIs will also go into `reading.go`

Reply:
Moved.